script.skin.helper.colorpicker/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+	<addon id="script.skin.helper.colorpicker" name="Skin Helper Service ColorPicker" version="1.0.13" provider-name="marcelveldt,sualfred">
+	<requires>
+		<import addon="xbmc.python" version="2.26.0"/>
+		<import addon="xbmc.addon" version="12.0.0"/>
```

Comment:
Remove xbmc.addon dependency, not needed

Reply:
thx, missed that.